Makefile

```diff
@@ -18,7 +18,7 @@ COVER_PROFILE = cover.out
 BIN_DIR := bin
 
 CRD_OPTIONS ?= "crd:trivialVersions=false,allowDangerousTypes=true,crdVersions=v1"
-KUSTOMIZE ?= go run -modfile=hack/tools/go.mod sigs.k8s.io/kustomize/kustomize/v3
+KUSTOMIZE ?= tools/bin/kustomize
```

Comment:
We're not being that consistent here by defining a variable for kustomize but not for controller-gen. I'm not sure how people were using this variable to override kustomize before, but I suspect that it will not work as expected with these changes. It's probably better to just not support customising it.
I think I'd be more tempted to only define `TOOLS_BIN_DIR = tools/bin`.